flutter/lib/src/sentry_flutter.dart

```diff
@@ -115,9 +123,18 @@ mixin SentryFlutter {
     // in errors.
     integrations.add(LoadReleaseIntegration(packageLoader));
 
+    if (options.platformChecker.hasNativeIntegration) {
+      integrations.add(MobileVitalsIntegration(nativeWrapper));
+    }
     return integrations;
   }
 
+  /// Manually set when you app finished startup. Make sure to set
+  /// [SentryFlutterOptions.autoAppStartFinish] to false on init.
+  static void setAppStartFinish(DateTime appStartFinish) {
+    _options?.appStartFinish = appStartFinish;
+  }
```

Comment:
Similar to https://github.com/getsentry/sentry-dart/pull/749/files#r816044870
Seems like a dedicated class would do better